chef/cookbooks/dhcp/recipes/default.rb

```diff
@@ -164,6 +164,6 @@
     end
   end
   supports :restart => true, :status => true, :reload => true
-  action :enable
+  action node[:provisioner][:enable_pxe] ? ["enable", "start"] : ["disable", "stop"]
```

Comment:
> After a mumble chat with @rhafer , provisioner::dhcp_update.rb and provisioner::update_nodes.rb are enabled again. Should be changed in the next release.

Just for completeness: The reason for keeping the update_nodes recipe running for now is, that it also does somethings that are needed when dhcp/pxe is disabled. E.g. when "forggetting" a node it is responsible for deleteing the chef node and role objects of the nodes. I rather not touch this before branching Tex.